For Beginners/180.md

```diff
@@ -0,0 +1,24 @@
+# User Inputs, Robot Checks
+
+Sometimes a user input is necessary. G1ANT.Robot allows to ask a user for some input with the `dialog.ask` command. And the `if` command is a perfect tool to check what was entered.
+
+Run this script:
+
+```G1ANT
+dialog.ask ‴Enter a digit (0-9):‴ result ♥input
+if ♥input=="1"
+   dialog ‴You entered 1‴
+  else if ♥input=="2"
+    dialog ‴You entered 2‴
+  else if ♥input=="3"
+    dialog ‴You entered 3‴
+  else
+    dialog ‴Your digit is greater than 3 or less than 1‴
```

Comment:
or is not a digit ;-)

Reply:
I know, I know - if you don't do what you're told